Documentation/kubernetes-on-aws-render.md

```diff
@@ -0,0 +1,255 @@
+# Configure your Kubernetes cluster on AWS
+
+This is the [second step of running Kubernetes on AWS][aws-step-1]. Before we launch our cluster, let's define a few parameters that the cluster requires.
+
+## Cluster parameters
+
+### EC2 key pair
+
+The keypair that will authenticate SSH access to your EC2 instances. A single key will be configured on each CoreOS node for debugging any issues you may encounter.
+
+Keep in mind that key pairs are tied to a region. More info in the [EC2 Keypair docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
+
+### External DNS name
+
+Select a DNS hostname where the cluster's API will be accessible. This information will first be used to provision the TLS certificate for the API server.
+
+When CloudFormation finishes creating your cluster, your controller will expose the TLS-secured API via a public IP address. You will need to create an A record for the DNS hostname which lists the IP address of the API. You can find this IP address later via `kube-aws status`.
+
+Optionally, an A record can be created in an existing route53 hosted zone. This is configured below.
+
+### KMS key
+
+[Amazon KMS](http://docs.aws.amazon.com/kms/latest/developerguide/overview.html) keys are used to encrypt and decrypt cluster TLS assets. If you already have a KMS Key that you would like to use, you can skip this step.
```

Comment:
s/this step/creating a new key and provide the Arn string for your existing key/